README.md

```diff
@@ -45,6 +45,56 @@ $ git clone https://github.com/okfn-brasil/notebooks.git
 
 We use serenata toolbox to download or generate the datasets. To install and use the toolbox just checkout the [toolbox's guide](https://github.com/okfn-brasil/serenata-toolbox#serenata-de-amor-toolbox).
 
+### Deploy in production with docker
+
+#### Prepare Docker Images
+
+- Create the docker images
+
+```bash
+docker build -t notebooks .
+```
+
+As we create the docker image, it will be stored in local docker images cache
+
+- Tag the image to maintain th version of the docker image
+
+```bash
+docker tag notebooks notebooks:1.0
+```
+
+*NOTE: By default when we create the docker image its default tag is latest. 0.1 is the version for current release*
+
+- Push docker image to dockerhub
+
+```bash
+# Syntax
+docker tag notebooks DOCKERHUB_USER_NAME/notebooks:VERSION
+
+# Example
+docker tag notebooks okbr/notebooks:0.1
+
+docker login
```

Comment:
Maybe you should add a note saying that the user must create an account in Docker Hub in order to successfully login in the CLI. A beginner user would be stuck wondering why his OS login doesn't work.